source/Jobbr.Server.WebAPI.Model/JobTriggerDtoBase.cs

```diff
@@ -5,11 +5,11 @@ namespace Jobbr.Server.WebAPI.Model
     /// <summary>
     /// The job trigger base.
     /// </summary>
-    public abstract class JobTriggerDtoBase
+    public class JobTriggerDtoBase
```

Comment:
Why is this class no longer considered abstract?

Reply:
After removing Newtonsoft, there was the following error when deserializing this type: "Deserialization of reference types without parameterless constructor is not supported" and removing the abstract keyword was the fix.

Reply:
Eh, I guess it's okay 😄